ourjapanlife/findadoc-server
# Add gitattributes file

Add a .gitattributes file to ensure consistent behavior across different environments (especially Windows and its preference for CRLF)


# What changed
- .gitattributes file based off of https://github.com/alexkaratarakis/gitattributes

# Testing instructions
- Need someone on a windows machine to try editing typescript files and ensure they are saved with LF not CRLF. Ensure there are no more husky errors like seen in #58
